djangoproject/templates/blog/entry_archive_day.html

```diff
@@ -1,12 +1,13 @@
 {% extends "base_weblog.html" %}
+{% load i18n %}
 
-{% block title %}{{ day|date:"F j" }} | Weblog{% endblock %}
+{% block title %}{{ day|date:"F j" }} | {% translate "Weblog" %}{% endblock %}
```

Comment:
As I mentionned in another PR, I think we should move to using named formats rather than hardcoded ones for localized strings:
```suggestion
{% block title %}{{ day|date:"MONTH_DAY_FORMAT" }} | {% translate "Weblog" %}{% endblock %}
```
(there's several instances of this in this file that need to be fixed too)